script.js

```diff
@@ -0,0 +1,123 @@
+	// Script
+    document.addEventListener('DOMContentLoaded', () => {
+        // Get the wrapper.
+            const block = document.querySelector('.q-and-a');
+    
+        // Get the video wrapper.
+            const video = block.querySelector('.q-and-a__video');
+            let player = null;
+    
+            // Initiate the video player.
+            if (video) {
+                player = new Plyr(video.querySelector('video'), {
```

Comment:
suggestion: The gate here, where you check for video is a great example of defensive programming. However, on LN12, there is another assumption where the first argument passed to `new Plyr` could return null. It would be good to set this up as a variable first and check for its existence.